.mozconfig

```diff
@@ -57,11 +57,11 @@ ac_add_options --disable-verify-mar
 ac_add_options --enable-application=browser
 #Crashes PPC64LE builds
 if test `uname -m` != ppc64le; then
-ac_add_options --enable-ccache=sccache
+ac_add_options --enable-ccache=ccache
 elif test `uname -m` = ppc64le; then
 ac_add_options --enable-ccache=ccache
 fi
-ac_add_options --enable-lto
+#ac_add_options --enable-lto
```

Comment:
Is this the way official Waterfox Current Linux builds are avoiding https://github.com/MrAlex94/Waterfox/issues/944 ?

Reply:
Yes that’s right - I’ve got a local mozconfig that’s slightly different